src/core/state.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+from typing import Dict
+
+
+class State:
+    """
+    This is a container class for all state variables as defined in this sheet: https://cornell.box.com/s/z20wbp66q0pseqievmadf515ucd971g2.
+    """
+
+    def __init__(self, state_dict: Dict = {}):
+        self.time = 0
+
+        # primitive state
+
+        # angular velocity
+        self.ang_vel_x = 0.0
+        self.ang_vel_y = 0.0
+        self.ang_vel_z = 0.0
+
+        # angular position (in quaternions)
```

Comment:
quaternions aren't a unit (I think they're unitless? ask ACS). A quaternion is a set of 4 numbers describing a rotation